Fix spacing/linking of s3 backup section (#2926)
CrunchyData/postgres-operator: docs/content/tutorial/backups.md

to upload to an S3 repository.

This method requires [additional setup in AWS IAM](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).

> Follow the procedure there for steps one and two:

Reviewer: ```suggestion
Use the procedure in the linked documentation for the first two steps described below:
```

> 1. Create an OIDC provider for your EKS cluster

Reviewer: ```suggestion
1. Create an OIDC provider for your EKS cluster.
```

[Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples/fork) repository:

1\. Add the `s3` section to the spec in `kustomize/s3/postgres.yaml` as discussed in the 

> `Using S3 Credentials` [section above](#using-s3-credentials). In addition to that, add the required `eks.amazonaws.com/role-arn` 

Reviewer 2: 🤔 I interpret the monospace formatting of backticks as code that should be taken literally. Having it on "Using S3 Credentials" is not what I would expect. It's literal, quotes of some kind make sense, so it's not confusing exactly.

🤔 A link on those words makes more sense to me, but the basis is entirely subjective. Do quotes belong around something like this? 🤷🏻 I'd love for us to use technical writing guidelines for things like this.

```suggestion
[Using S3 Credentials](#using-s3-credentials) section above. In addition to that, add the required `eks.amazonaws.com/role-arn` 
```

Author: Whoops, sorry about that, missed this when merging, hold on.
